### PR TITLE
Code update

### PR DESCRIPTION
**Pull request by Matejk00**

### CODE REVIEW

File: feature-libs/organization/user-registration/components/form/user-registration-form.component.html [hunk 1]
```diff
@@ -1,285 +1,277 @@
...
-              />
-            </label>
-          </div>
+    <ng-container
+      *ngIf="countries$ | async as countries"
+      formGroupName="country"
+    >
+      <div>
+        <div *ngIf="countries.length !== 0">
+          <label>
```
Comment (Pucek9) [inline feature-libs/organization/user-registration/components/form/user-registration-form.component.html]:
```suggestion
      <label *ngIf="countries.length !== 0">          
```

File: feature-libs/organization/user-registration/components/form/user-registration-form.component.html [hunk 2]
```diff
@@ -1,285 +1,277 @@
...
-                }}"
-                formControlName="lastName"
-              />
-              <cx-form-errors
-                [control]="registerForm.get('lastName')"
-              ></cx-form-errors>
-            </label>
-          </div>
+  <form [formGroup]="registerForm" (ngSubmit)="submit()">
+    <div>
```
Comment (Pucek9) [inline feature-libs/organization/user-registration/components/form/user-registration-form.component.html]: are they necessary? 😅
```suggestion
```

File: feature-libs/organization/user-registration/components/form/user-registration-form.component.html (hunk 1 above)
Comment (Matejk00) [reply feature-libs/organization/user-registration/components/form/user-registration-form.component.html]: done

File: feature-libs/organization/user-registration/components/form/user-registration-form.component.html (hunk 2 above)
Comment (Matejk00) [reply feature-libs/organization/user-registration/components/form/user-registration-form.component.html]: removed :)

**Pull request merged**